### ISSUE TITLE
Consider development mode logging frequency

### ISSUE THREAD

**Issue by davidmytton** [arcjet/arcjet-js#1781]
A user reported that the `✦Aj WARN Using 127.0.0.1 as IP address in development mode` warning is annoying because it pollutes the logs when you're trying to find other things to focus on. Perhaps a compromise is to log it once during client init?

<img width="481" alt="logs" src="https://github.com/user-attachments/assets/38620001-4edc-459d-8dae-641accf1813e">

**Comment by blaine-arcjet** [comment]
@davidmytton I can move the development mode warning but I think I'm going to leave the production mode warning (see below) to log each time because we don't know if the IP is missing until a request comes in.

While I could set a "has logged" flag, I think this warning is a big enough problem to "be annoying" about.

<img width="787" alt="Image" src="https://github.com/user-attachments/assets/eb896fc1-38b1-46e1-952a-ffe002a1160c" />

**Comment by davidmytton** [comment]
Agreed. We want to notify users if the client IP is missing in production (or if it's a non-public IP) because that should be an unusual (error) case. It's the opposite in dev, so reducing the logging verbosity makes sense there.
